src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -33,6 +33,15 @@ public static class ILoggerExtensions
             + ContextProperties.DependencyTracking.IsSuccessful + "} - Context: {@"
             + ContextProperties.EventTracking.EventContext + "})";
 
+        private const string DependencyWithoutDataFormat =
```

Comment:
Added this bc I didn't come up with a dependency data value. If you think it's usable, we could add also some kind of "operation name" like the SQL dependency to set as dependency data. Then we could reuse the dependency data format.

Reply:
From the looks of it they use the account name as the target so we should pass the table name as data
![image](https://user-images.githubusercontent.com/4345663/80204724-acda7300-8629-11ea-84c0-e3306dccdd50.png)
https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-map?tabs=net#composite-application-map

Reply:
Yes, done that.

Reply:
Waiting for tests to complete and see if it's improved now

Reply:
Should have `Remote dependency name` 🤞